### PR TITLE
Updating myTarget iOS Adapter.

### PR DESCRIPTION
**Pull request by aserrr**
Fixed minor bug in Native adapter (disable images loading)
Supported parameters in Extras.
Supported Extras in Rewarded ads.

### CODE REVIEW

File: adapters/MyTarget/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 ## myTarget iOS Mediation Adapter Changelog
 
+#### [Version 5.14.2.1](In progress)
```
Comment (chlandacastro) [inline adapters/MyTarget/CHANGELOG.md:3]: Could you remove the brackets like so,  #### Version 5.14.2.1 (In progress)    
Also, the new myTarget iOS SDK is v5.14.3.   so the new Adapter version 5.14.3.0

Comment (aserrr) [reply adapters/MyTarget/CHANGELOG.md]: Ok, no probs.

File: adapters/MyTarget/MyTargetAdapter/GADMAdapterMyTargetNative.m [hunk 2]
```diff
@@ -148,15 +148,15 @@ - (void)getNativeAdWithAdTypes:(nonnull NSArray<GADAdLoaderAdType> *)adTypes
         (GADNativeAdImageAdLoaderOptions *)adLoaderOptions;
     if (imageOptions.disableImageLoading) {
       _autoLoadImages = NO;
-      cachePolicy = cachePolicy & ~MTRGCachePolicyImages;
+      cachePolicy = MTRGCachePolicyVideo;
```
Comment (joshh-devrel) [inline adapters/MyTarget/MyTargetAdapter/GADMAdapterMyTargetNative.m:151]: Question: May I ask why the cache policy is `MTRGCachePolicyVideo` if the image loading option is disabled?

File: adapters/MyTarget/MyTargetAdapter/GADMAdapterMyTargetConstants.m [hunk 3]
```diff
@@ -14,7 +14,7 @@
 
 #import "GADMAdapterMyTargetConstants.h"
 
-NSString *const _Nonnull kGADMAdapterMyTargetVersion = @"5.14.2.0";
+NSString *const _Nonnull kGADMAdapterMyTargetVersion = @"5.14.2.1";
```
Comment (joshh-devrel) [inline adapters/MyTarget/MyTargetAdapter/GADMAdapterMyTargetConstants.m:17]: Should be "5.14.3.0" instead.

File: adapters/MyTarget/MyTargetAdapter/GADMAdapterMyTargetExtras.m [hunk 4]
```diff
@@ -24,4 +26,24 @@ - (nonnull instancetype)init {
   return self;
 }
 
+- (void)setParameter:(nullable NSString *)parameter forKey:(nonnull NSString *)key {
```
Comment (joshh-devrel) [inline adapters/MyTarget/MyTargetAdapter/GADMAdapterMyTargetExtras.m:29]: Question: Is there a list that shows what parameters are supported?

File: adapters/MyTarget/MyTargetAdapter/GADMAdapterMyTargetNative.m (hunk 2 above)
Comment (aserrr) [reply adapters/MyTarget/MyTargetAdapter/GADMAdapterMyTargetNative.m:151]: Default cache policy is ALL (Images + Video), so if Images loading disabled, then Video should stay enabled.

File: adapters/MyTarget/MyTargetAdapter/GADMAdapterMyTargetExtras.m (hunk 4 above)
Comment (aserrr) [reply adapters/MyTarget/MyTargetAdapter/GADMAdapterMyTargetExtras.m:29]: Actually there is no such list. We allow users to set [String:String] dictionary as custom parameters with no additional requirements to Key string.

**Pull request merged**